build-tools/automation/azure-pipelines.yaml

```diff
@@ -493,6 +477,210 @@ stages:
         targetPath: $(Build.ArtifactStagingDirectory)
       condition: always()
 
+  # Check - "Xamarin.Android (Test MSBuild Mac)"
+  - job: mac_msbuild_tests
+    displayName: MSBuild Mac
+    pool: $(XA.Build.Mac.Pool)
+    timeoutInMinutes: 180
+    cancelTimeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+    - template: yaml-templates/mac/setup-test-environment.yaml
+
+    - task: DownloadPipelineArtifact@1
+      inputs:
+        artifactName: $(NUnitTestAssembliesArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)
+
+    - script: >
+        mono packages/NUnit.ConsoleRunner.$(NUnitConsoleVersion)/tools/nunit3-console.exe
```

Comment:
Does this line and the following line need a `\` line continuation?  Or are all the lines "joined" into one line before execution?

Reply:
When using a `>` in a multi-line string in YAML, new lines will be replaced by spaces.